OTAnalytics/plugin_ui/customtkinter_gui/dummy_viewmodel.py

```diff
@@ -662,8 +662,11 @@ def _update_metadata(self, selected_section: Section) -> None:
         self._finish_action()
```

Comment:
When editing a section, clicking on the `Cancel` button will raise an `CancelAddSection` error. See l.651 and l.107 in `toplevel_sections.py`

Reply:
This got fixed in another issue. It is merged into main and therefore fixed in this branch.